### ISSUE TITLE
Min opens to a new, empty task

### ISSUE THREAD

**Issue by clayton-grey** [minbrowser/min#1387]
-1.17.1
- macOS 10.14.6

## Expected Behavior
Min opens to the previously open tab and task.

## Actual Behavior
Min opens to a new, empty task.

## To Reproduce
Open Min - though it seems somewhat inconsistent. It appears that if I quickly close and re-open the application, it behaves as expected. If I leave the application closed for some (unknown) duration or condition, it seems to return to the "Open with new, empty task" behavior. Will add more if I find any more specificity.

**Comment by PalmerAL** [comment]
This is mostly intended behavior, although what's supposed to happen is that a "return to your last task" button appears right below the search bar. Does that not show up for you?

**Comment by clayton-grey** [comment]
I don't recall seeing such a thing. I will try to watch for it next time the behavior happens. 

Is there a preference flag to disable the new task on open? I, personally, never want that. I use Min for specific topical browsing and make new Tasks only with specific purposes in mind, and it's rare that I add them.

**Comment by clayton-grey** [comment]
Okay. I can verify that I see the indicated UI, but because it blends in I had never "seen" it. It appears like standard auto complete UI, which isn't what I am expecting to see on launch.

**Comment by dimpogkas** [comment]
Have you found a way to disable it? I think it shouldn't be there in the first place...

**Comment by clayton-grey** [comment]
No. As noted, it seems to be their intended design, though I still wish it wasn't. I just find I close it less often, even when I'm not using it.
